lib/builder.js

```diff
@@ -264,7 +264,7 @@ function executeConfigFile(saveForReset, ignoreBaseURL, source) {
   };
 
   // jshint evil:true
-  new Function(source.toString()).call(global);
+  eval(source.toString());
```

Comment:
I see. Somehow the code must be executed in a way so that

     something = 'some value'; 

did not pollute global scope, but

    System.config(...);

referred to the local System declared in executeConfigFile.

I don't see any easy way to achieve both. Since there is transpiler available, one way to do it would be to call javascript parser, walk the AST and patch calls to System.config() to refer to the local System (which needs to be made accessible somehow), then generate source and call new Function().call() on it. But that seems like an overkill.

I'm happy to have this as wontfix, actually - I know how to use this API properly now, and parallel builds do not matter that much for me.